modules/gapi/include/opencv2/gapi/core.hpp

```diff
@@ -501,6 +504,42 @@ namespace core {
             return in.withType(in.depth, in.chan).withSize(dsize);
         }
     };
+
+    G_TYPED_KERNEL(GParseSSDWL, <GDetections(GMat, GOpaque<Size>, float, int)>, "org.opencv.core.parseSSDWL") {
+        static std::tuple<GArrayDesc,GArrayDesc> outMeta(const GMatDesc&, const GOpaqueDesc&, float, int) {
+            return std::make_tuple(empty_array_desc(), empty_array_desc());
+        }
+    };
+
+    G_TYPED_KERNEL(GParseSSD, <GRects(cv::GMat, cv::GOpaque<cv::Size>, float, bool)>, "org.opencv.core.parseSSD") {
+        static cv::GArrayDesc outMeta(const cv::GMatDesc &, const cv::GOpaqueDesc &, float, bool) {
```

Comment:
```suggestion
        static cv::GArrayDesc outMeta(const cv::GMatDesc&, const cv::GOpaqueDesc&, float, bool) {
```

Reply:
Fixed. M B.

Reply:
Yes, check this please one more time

Reply:
Checked.

Reply:
This particular string is not correct :) please, remove spaces before &

Reply:
Hah, fixed.